packages/api-react/src/services/harvester.ts

```diff
@@ -1,10 +1,12 @@
 import { Harvester } from '@chia-network/api';
 
+import api, { baseQuery } from '../api';
+import onCacheEntryAddedInvalidate from '../utils/onCacheEntryAddedInvalidate';
 import { query, mutation } from '../utils/reduxToolkitEndpointAbstractions';
 import { apiWithTag } from './farmer';
 
 const apiWithTag2 = apiWithTag.enhanceEndpoints({
-  addTagTypes: ['Plots', 'PlotDirectories'],
+  addTagTypes: ['Plots', 'PlotDirectories', 'harvesterConfig'],
```

Comment:
nit: Can we keep the capitalization consistent? `harvesterConfig` vs `HarvesterConfig`